RCTAppleHealthKit/Swift/RCTAppleHealthKit+Steps.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  RCTAppleHealthKit.swift
+//  RCTAppleHealthKit
+//
+//  Copyright © 2024 Bearable. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import CoreLocation
```

Comment:
Just noticed this - looks like it's been imported by mistake?

This *could* trigger a rejection because it will require app location permissions, so should take it out.

Reply:
Thanks for catching this! I'll update it now 👍